README.md

```diff
@@ -9,63 +9,65 @@ It is based on the [Interaction-Calculus](https://github.com/VictorTaelin/Intera
 
 Currently Bend only supports strict/eager evaluation. If you need lazy, optimal evaluation, we recommend using [HVM1](https://github.com/HigherOrderCO/HVM1) for now.
 
-
 ## Installation
 
 With the nightly version of rust installed, clone the repository:
+
 ```bash
 git clone https://github.com/HigherOrderCO/bend.git
 
 cd bend
 ```
 
 Install using cargo:
+
 ```bash
 cargo install --path . --locked
 ```
 
 If you want to run programs directly from Bend, you also need to have [HVM](https://github.com/HigherOrderCO/hvm2) installed.
 
-
 ## Usage
 
-Command   | Usage                 | Description
---------- | --------------------- | ---
-Check     | `bend check <file>`   | Checks if a program is valid
-Compile   | `bend compile <file>` | Compiles a program to HVM and outputs it to stdout
-Run       | `bend run <file>`     | Compiles and then runs a program in HVM
-Normalize | `bend norm <file>`    | Compiles and then normalizes a program in HVM, outputting the result to stdout
-Desugar   | `bend desugar <file>` | Desugars a program to the core syntax and outputs it to stdout
-
+| Command   | Usage                 | Description                                                                    |
+| --------- | --------------------- | ------------------------------------------------------------------------------ |
+| Check     | `bend check <file>`   | Checks if a program is valid                                                   |
+| GenHvm    | `bend gen-hvm <file>` | Compiles a program to HVM and outputs it to stdout                             |
+| Run       | `bend run <file>`     | Compiles and then runs a program with the Rust HVM implementation              |
+| Run-C     | `bend run-c <file>`   | Compiles and then runs a program with the C HVM implementation                 |
+| Run-Cu    | `bend run-cu <file>`  | Compiles and then runs a program with the Cuda HVM implementation              |
+| Normalize | `bend norm <file>`    | Compiles and then normalizes a program in HVM, outputting the result to stdout |
```

Comment:
Add the norm-cu command here. Also, change `norm` to `norm-c`. I havent checked if the rust runtime has IO, but if it does then add that command  too

Reply:
The Rust impl does not accept the `--io` argument

Reply:
Ok

Reply:
Then we should remove `bend run` and add `bend norm-c`